.github/workflows/ci.yaml

```diff
@@ -13,13 +13,23 @@ jobs:
     strategy:
       fail-fast: false
     steps:
+    - name: Extract pull request number from PR description
+      id: extract_analyzer_pull_request_number
+      run: |
+        PULL_REQUEST_NUMBER=$(echo "${{ github.event.pull_request.body }}" | grep -oP '[A|a]nalyzer.?[P|p][R|r]: \K\d+' || true)
+        if [ -z "$PULL_REQUEST_NUMBER" ]; then
+          echo "::set-output name=ref::main"
+        else
+          echo "::set-output name=ref::refs/pull/$PULL_REQUEST_NUMBER/head"
```

Comment:
should we grab `/merge` instead?

Reply:
@fabianvf just thought about the case where two PRs affect each other enough that the analyzer is not merged